superset/assets/src/visualizations/nvd3/utils.js

```diff
@@ -46,14 +46,18 @@ export function generateRichLineTooltipContent(d, valueFormatter) {
     + '</td></tr></thead><tbody>';
   d.series.sort((a, b) => a.value >= b.value ? -1 : 1);
   d.series.forEach((series) => {
+    let key = dompurify.sanitize(series.key);
```

Comment:
This exact same code is repeated twice in this small PR, let's refactor that simple logic into a function.